salt/modules/ddns.py

```diff
@@ -191,16 +202,25 @@ def update(zone, name, ttl, rdtype, data, nameserver='127.0.0.1', timeout=5,
                     is_exist = True
                     break
 
+    log.debug('record exists: %s', is_exist)
+
     dns_update = dns.update.Update(zone, keyring=keyring, keyname=keyname,
                                    keyalgorithm=keyalgorithm)
     if replace:
+        log.info('Replacing record %s', name)
         dns_update.replace(name, ttl, rdata)
     elif not is_exist:
+        log.info('Creating record %s', name)
         dns_update.add(name, ttl, rdata)
     else:
+        log.info('Not doing anything, record exists and in correct state', name)
```

Comment:
There is no string format for `name` here.

Reply:
Now there is!